File: backend/src/main/java/com/shx/locacao/veiculos/model/Customer.java
package com.shx.locacao.veiculos.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDate;


@Entity
public class Customer implements BaseEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private Long cpf;
    private String name;
    private LocalDate birthdate;
    private Boolean isActive;

    public Customer(){}

    public Customer(Integer id, Long cpf, String name, LocalDate birthdate, Boolean isActive) {
        this.id = id;
        this.cpf = cpf;
        this.name = name;
        this.birthdate = birthdate;
        this.isActive = isActive;
    }

    @Override
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Long getCpf() {
        return cpf;
    }

    public void setCpf(Long cpf) {
        this.cpf = cpf;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public LocalDate getBirthdate() {
        return birthdate;
    }

    public void setBirthdate(LocalDate birthdate) {
        this.birthdate = birthdate;
    }

    public Boolean isActive() {
        return isActive;
    }

    public void setActive(Boolean isActive) {
        this.isActive = isActive;
    }
}

File: web/scripts/model/Customer.js
class Customer{
    constructor(id, cpf, name, birthdate, active){
        this.id = id;
        this.cpf = cpf;
        this.name = name;
        this.birthdate = birthdate;
        this.active = active;
    }

    getId (){
        return this.id;
    }

    setId (id){
        this.id = id;
    }

    getCpf (){
        return this.cpf;
    }

    setCpf (cpf){
        this.cpf = cpf;
    }

    getName (){
        return this.name;
    }

    setName (name){
        this.name = name;
    }

    
    getBirthdate (){
        return this.birthdate;
    }

    setBirthdate (birthdate){
        this.birthdate = birthdate;
    }   

    getActive (){
        return this.active;
    }

    setActive (active){
        this.active = active;
    } 
   

}